src/models/multiChart.js

```diff
@@ -501,6 +532,11 @@ nv.models.multiChart = function() {
         noData:    {get: function(){return noData;}, set: function(_){noData=_;}},
         interpolate:    {get: function(){return interpolate;}, set: function(_){interpolate=_;}},
         legendRightAxisHint:    {get: function(){return legendRightAxisHint;}, set: function(_){legendRightAxisHint=_;}},
+        showY1Axis:    {get: function(){return showY1Axis;}, set: function(_){showY1Axis=_;}},
+        showY1AxisZero:    {get: function(){return showY1AxisZero;}, set: function(_){showY1AxisZero=_;}},
+        showY2Axis:    {get: function(){return showY2Axis;}, set: function(_){showY2Axis=_;}},
+        showY2AxisZero:    {get: function(){return showY2AxisZero;}, set: function(_){showY2AxisZero=_;}},
+        stackBars:    {get: function(){return stackBars;}, set: function(_){stackBars=_;}},
```

Comment:
this isn't really doing much, might as well let them just call chart.bars.stacked(true)